docs/getting-started/configure-app-with-plugins.md

```diff
@@ -4,17 +4,22 @@ title: Configuring App with plugins
 description: Documentation on How Configuring App with plugins
 ---
 
+Backstage plugins customize the app for your needs. There is a
+[plugin marketplace](https://backstage.io/plugins) with plugins for many common
+infrastructure needs - CI/CD, monitoring, auditing, and more.
+
 ## Adding existing plugins to your app
 
-The following steps assume that you have created a new Backstage app and want to
-add an existing plugin to it. We are using the
+The following steps assume that you have
+[created a Backstage app](./create-an-app.md) and want to add an existing plugin
+to it. We are using the
 [CircleCI](https://github.com/backstage/backstage/blob/master/plugins/circleci/README.md)
 plugin in this example.
 
 1. Add the plugin's npm package to the repo:
 
 ```bash
-yarn add @backstage/plugin-circleci
+yarn workspace app add @backstage/plugin-circleci
```

Comment:
Thanks! Yeah this needed an update. On a sidenote, I've seen people being confused about adding plugins and packages to the root of the repository and encountering lerna error ([Recent example](https://github.com/backstage/backstage/issues/4774)) and not knowing that frontend/backend packages should be installed separately in `packages/app` and `packages/backend`.

Do you think this is the best place to document this - that since this is a Backstage frontend plugin, we will add it inside `packages/app`? Usually each plugin (e.g. [TechDocs](https://backstage.io/docs/features/techdocs/getting-started)) has its own guide where they highlight this behavior.

Reply:
Sounds good, and perhaps for the even more general case we could find some external documentation about monorepos that we could link out to for a deep dive? Just a quick couple of mentions of the most important aspects but with a link for a [deep](https://classic.yarnpkg.com/en/docs/workspaces/) [dive](https://lerna.js.org/)

On a completely separate note, I wonder if we should look into dropping lerna, as we're really only using it to execute commands in multiple packages, and there might be some slimmer tooling for that purpose :grin:

Reply:
The cli uses `@lerna/project` and `@lerna/package-graph`, are those easily replaceable?

Reply:
Yep, I'm thinking whatever we'd use could provide that, and it's not too much work if we want to make it a custom thing either. It's basically just finding all `package.json` in the workspace and parsing them, the graph bits we don't really use. We do also shell out to lerna in the CLI, and those are probably the worst bits to replace

Reply:
@OrkoHunter Good point, I added a small note for this doc at least 👍